pkg/api/aim/dashboards.go

```diff
@@ -22,10 +22,15 @@ func GetDashboards(c *fiber.Ctx) error {
 
 	var dashboards []database.Dashboard
 	if err := database.DB.
-		Preload("App").
-		Joins("LEFT JOIN apps ON apps.id = dashboards.app_id").
+		InnerJoins(
+			"App",
+			database.DB.Select(
+				"ID", "Type",
+			).Where(
+				`"App".namespace_id = ? AND NOT "App".is_archived`, ns.ID,
```

Comment:
why the quoting?

Reply:
otherwise  it is just doesn't work. look at this:
<img width="1773" alt="Screenshot 2023-09-15 at 13 54 55" src="https://github.com/G-Research/fasttrackml/assets/7852635/bca2253c-8737-401e-a213-a80fb813c799">

Reply:
I see, then I would recommend this:

```go
		InnerJoins("App",
			database.DB.
				Select("ID", "Type").
				Where(&database.App{NamespaceID: ns.ID}, "NamespaceID", "IsArchived")).
		Where("NOT dashboards.is_archived").
		Order("dashboards.updated_at").
		Find(&dashboards).
```